Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Tuxemon/Tuxemon](https://hosted.weblate.org/projects/tuxemon/tuxemon/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/tuxemon/-/tuxemon/horizontal-auto.svg)

LINUX-SAUNA committed:
Translated using Weblate (Finnish)

Currently translated at 16.7% (237 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/fi/

Lisapple committed:
Translated using Weblate (French)

Currently translated at 90.0% (1271 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/fr/

Lisapple committed:
Translated using Weblate (French)

Currently translated at 92.2% (1303 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/fr/

kerizane commented:
Slight issue, the values within braces shouldn't have been translated.

E.g. the following
msgid "item_no_available_target"
msgstr "{nimi} ei voida käyttää yhteenkään Tuxemoniisi tällä hetkellä."

Should be:
msgid "item_no_available_target"
msgstr "{name} ei voida käyttää yhteenkään Tuxemoniisi tällä hetkellä."

kerizane reviewed (changes requested):
Fix translations

tobim91 committed:
Translated using Weblate (German)

Currently translated at 33.0% (466 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/de/

EddieMattos committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 4.3% (62 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/pt_BR/

kerizane committed:
Translated using Weblate (Finnish)

Currently translated at 16.7% (236 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/fi/

EddieMattos committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 28.1% (398 of 1412 strings)

Translation: Tuxemon/Tuxemon
Translate-URL: https://hosted.weblate.org/projects/tuxemon/tuxemon/pt_BR/

kerizane commented:
I fixed the Finnish translations on weblate, not sure if they will be pushed automatically...
